presto-benchto-benchmarks/generate_schemas/generate-tpcds.py

```diff
@@ -47,7 +47,7 @@
     else:
         raise ValueError(new_schema)
 
-    print 'CREATE SCHEMA hive.%s;' % (new_schema,)
+    print('CREATE SCHEMA hive.{};'.format(new_schema,))
```

Comment:
i guess for py 3 just adding `(...)` for `print` would be sufficient?

i don't mind the switch to .format, but ideally it would go as a separate change...  

since when string#format is supported?

Reply:
`str.format()` has been supported from python 2.6.
https://docs.python.org/2.7/library/stdtypes.html#str.format